src/graphql/HolofuelRefundAllDeclinedMutation.gql

```diff
@@ -0,0 +1,14 @@
+mutation HolofuelRefundAllDeclined ($listOfDeclinedTransactions:[CancelDeclinedTransactionsInput]) {
```

Comment:
see comment on `schema.graphql` about this

Reply:
I'll update the name here. :+1: